src/Surfnet/ServiceProviderDashboard/Domain/Repository/EntityChangeRequestRepository.php

```diff
@@ -29,7 +29,12 @@ interface EntityChangeRequestRepository
     /**
      * Open an entity change request in manage
      */
-    public function openChangeRequest(ManageEntity $entity, ?ManageEntity $pristineEntity, Contact $contact): array;
+    public function openChangeRequest(
+        ManageEntity $entity,
+        ?ManageEntity $pristineEntity,
+        Contact $contact,
+        string $jiraTicketNumber
```

Comment:
I would suggest a comma on the last one as well to prevent chage request on that line when another param is added.

There is a cs rule for that